### PR TITLE
Feature button feedback

### PR DESCRIPTION
**Pull request by pochretien**

### CODE REVIEW

**Comment by wfortin** [comment]
:shipit:

File: scss/elements/btn.scss [hunk 1]
```diff
@@ -86,4 +86,73 @@
...
+
+    color: transparent;
+
+    outline: 0;
+
+    user-select: none;
+
+    &:before {
+      position: absolute;
+      top: 40px;
```
Comment (GermainBergeron) [inline scss/elements/btn.scss]: Should it be `$button-height + x?` where X is the margin we want?

File: scss/elements/btn.scss [hunk 2]
```diff
@@ -86,4 +86,73 @@
...
+      right: 0;
+    }
+
+    &.mod-confirm-left:before {
+      left: 0;
+    }
+
+    .confirm-icon {
+      position: absolute;
+      top: calc(50% - #{$button-confirm-icon-width}/2);
```
Comment (GermainBergeron) [inline scss/elements/btn.scss]: You should put the ` / 2` between the curly braces

**Comment by JPLachance** [comment]
Fix the build than ship it!

**Pull request merged**